app/assets/javascripts/patients.coffee

```diff
@@ -23,6 +23,23 @@ valueToNumber = (val) ->
   +val || 0
 
 ready = ->
+  fields = [
```

Comment:
Can we rename this variable a bit for clarity's sake? Not sure I'd know what this did just by looking at it if I saw it out outside the context of this PR.